core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java

```diff
@@ -378,11 +379,7 @@ public Boolean call() throws AlluxioTException, TException {
         }
       });
     } catch (AlluxioException e) {
-      if (e instanceof WorkerOutOfSpaceException) {
-        return false;
-      } else {
-        throw new IOException(e);
-      }
+      throw new IOException(e);
```

Comment:
nvm, I saw you handle `WorkerOutOfSpaceException` on the worker side. So the semantics should be the same.